### DISCUSSION TITLE
if there is change in one repo  it should trigger the build in another repo

### DISCUSSION THREAD

**Discussion by SowmyaPhilips** [community/community#36615] [Other Feature Feedback, Questions, & Ideas]
can any one please suggest how to trigger a build in one repo if there is change another repo

**Comment by airtower-luna** [comment]
There's nothing built-in. If you have a token with appropriate permission in the secrets for the repository that changed, you can [send a `workflow_dispatch` event](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch) to the other repository.

**Comment by ossanna16** [comment]
Hey there @SowmyaPhilips and thank you for asking a good question! I'm glad you found our community 🙂
